# feature map
# header 
Header header
# circle x position of center
float64[] cx
# circle y position of center
float64[] cy
# circle radius
float64[] r
